pizzaplace/models.py

```diff
@@ -1,8 +1,23 @@
 from django.db import models
+from model_utils.models import TimeStampedModel
+from django.core.validators import RegexValidator
+from pizzaplace.services.yelp_api import YelpApi
+from django.core.exceptions import ValidationError
 
-# Create your models here.
-class PizzaPlace(models.Model):
+def validate_url(link):
+  validator = RegexValidator(
+    regex='yelp\.com\/biz\/\w+(-\w+)*(\?(.)*)?',
+    message="Url should be in form 'https://www.yelp.com/biz/some-pizza-place'",
+    code='invalid_url')
+  return validator(link)
+
+def validate_yelp_business_exists(link):
+  if not YelpApi(link).url_exists():
```

Comment:
Now we're getting somewhere! This collaborator object encapsulates all the details about how this business can exist, and this function just has to deal with the validation. :+1: